.github/workflows/pr.yml

```diff
@@ -37,8 +41,30 @@ jobs:
         run: ./build.sh publishbinaries
         
       - uses: actions/upload-artifact@v4
+        if: matrix.os == 'ubuntu-latest'
         with:
           name: docs-builder-binary
           path: .artifacts/publish/docs-builder/release/docs-builder
           if-no-files-found: error
           retention-days: 1
+
+  build-windows:
+    runs-on: windows-latest
+    strategy:
```

Comment:
this matrix strategy makes no sense we should remove it.

Reply:
Should we revert to https://github.com/elastic/docs-builder/pull/425/commits/1722712e90ae19f89087127b93fe3a260161d230 ?

Reply:
Ah there are different steps for windows.

Reply:
Ah, nvm. Got it

Reply:
https://github.com/elastic/docs-builder/pull/425/commits/61d7e599063d0ad481be56cf89d2e6231177377c